src/request/export/Export.js

```diff
@@ -24,7 +24,35 @@ class Export {
         this._exportWordsHandler = new ExportWords(request, this._wordsSucceeded.bind(this), this._reject.bind(this));
         this._exportFileHandler = new ExportFile(request, this._fileSucceeded.bind(this), this._reject.bind(this));
 
-        this._build();
+        /** @type {HTMLElement} */
+        const $moreExportOptionsPage = (
+            document.getElementById(Export.Pages.MORE_EXPORT_OPTIONS)
+                ? document.getElementById(Export.Pages.MORE_EXPORT_OPTIONS)
+                : this._buildMoreExportOptions()
+        );
+        /** @type {HTMLElement} */
+        const $keyfilePage = (document.getElementById('download-key-file'));
+
+        /** @type {HTMLElement} */
+        const $finishRequestButton = ($moreExportOptionsPage.querySelector('.finish-request'));
+        /** @type {HTMLElement} */
+        this.$wordsButton = ($moreExportOptionsPage.querySelector('.go-to-words'));
+        /** @type {HTMLElement} */
+        this.$fileButton = ($moreExportOptionsPage.querySelector('.go-to-file'));
+        /** @type {HTMLElement} */
+        const $wordsButton = ($keyfilePage.querySelector('.go-to-words'));
+
+        $finishRequestButton.addEventListener('click', () => {
+            this._resolve({ success: true });
+        });
+        $wordsButton.addEventListener('click', () => this._exportWordsHandler.run());
+        this.$wordsButton.addEventListener('click', () => this._exportWordsHandler.run());
```

Comment:
Is line 48 accidentally left in and should be removed?

Reply:
Same in `ExportFile.js` and `ExportWords.js`.

Reply:
there are 2 Buttons that lead to words. the first one is on the first page and the second one on the last. So both of them are needed. I renamed the first for clarity.

What do you mean by 
'Same in `ExportFile.js` and `ExportWords.js`.